src/microbe_stage/editor/action_data/BehaviourActionData.cs

```diff
@@ -46,4 +48,16 @@ protected override CombinableActionData CombineGuaranteed(CombinableActionData o
 
         return new BehaviourActionData(behaviourChangeActionData.NewValue, OldValue, Type);
     }
+
+    protected override void MergeGuaranteed(CombinableActionData other)
+    {
+        var behaviourChangeActionData = (BehaviourActionData)other;
```

Comment:
Would it be possible to leverage the existing combining code? Even if this was changed so that MergeGuaranteed needed to return a CombinableActionData, I think that would be better than every action needing to program the dynamic MP merging *and* this merging method.

Reply:
Can't figure out how.

Reply:
Okay so I couldn't either, but I found a pretty major not handled case, rigidity changes in subsequent multicellular cell edits, which have this kind of really ~~nasty~~ nice structure:

![Kuvakaappaus - 2022-06-14 18-42-48](https://user-images.githubusercontent.com/3796411/173622745-93aa1608-b76c-441f-8a7a-581ba168bbeb.png)

So let me know what you think. If no one can suggest a further simplified design, I'm happy enough with the code now.

Reply:
I don't even know there's such data in CombinedEditorAction.